build/annotate-clients.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
```

Comment:
This file should at least have a comment to describe what it does and the usage.